src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -36,135 +36,143 @@ <h1 class="h1" id="title">
     <p class="p">{{ object.summary }}</p>
     {{ object|product_ckeditor_content|safe }}
 
-    {% if object.question_set.exists %}
+    {% if object.question_set.exists or object.files.exists or object.conditions.exists or object.locations.exists or product_links.exists or object.related_products.published.exists or object.contacts.exists %}
```

Comment:
@Bartvaderkin Yes it's not pretty - i thought: since this is based on data coming in or not, it needs to be done in the template (and not with javascript).